dev/com.ibm.ws.jaxws.tools/src/com/sun/tools/ws/wscompile/WsimportTool.java

```diff
@@ -503,10 +554,10 @@ protected boolean compileGeneratedClasses(ErrorReceiver receiver, WsimportListen
             args.add("-d");
             args.add(classDir);
             args.add("-classpath");
-            args.add(classpathString);
+            args.add(classpathString + File.pathSeparator + options.classpath);
```

Comment:
should we only append options.classpath if java version is >= 9 in order to maintain existing behavior?

Reply:
Yes, updates will be part of next commit.